jans-lock/cedarling/tests/fetch-schema.rs

```diff
@@ -1,8 +1,14 @@
-use cedarling::utils;
+use cedar_policy::*;
 use wasm_bindgen_test::*;
 
 #[wasm_bindgen_test]
 async fn test() {
-	let schema = utils::fetch_schema().await;
+	let schema = include_str!("../policy-store/schema.txt");
```

Comment:
This is an empty file, and is not testing anything.

Reply:
Not anymore, I just pushed a commit that read the policy store as JSON